modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -67,31 +67,34 @@ The `--copy-network` option only copies networking configuration found under `/e
-+
 [IMPORTANT]
 ====
-For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
+The use of a custom partition scheme on your node could result in {product-title} not monitoring or alerting on some node partitions. If you are overriding the default partitioning, see link:https://access.redhat.com/articles/4766521[Understanding OpenShift File System Monitoring (eviction conditions)] for more information about how {product-title} monitors your host file systems.
 ====
-+
+
+{product-title} monitors the following two filesystem identifiers:
+
+* `nodefs`, which is the filesystem that contains `/var/lib/kubelet`
+* `imagefs`, which is the filesystem that contains `/var/lib/containers`
+
+For the default partition scheme, `nodefs` and `imagefs` monitor the same root filesystem, `/`.
+
+If you want to override the default partitioning when installing {op-system} on an {product-title} cluster node, you must create separate partitions.
+
 [IMPORTANT]
 ====
-Kubernetes supports only two file system partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
+For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
 ====
 
-* Retaining existing partitions: For a brownfield installation where you are reinstalling {product-title} on an existing node and want to retain data partitions installed from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions.
+You might want to add a separate storage partition for your containers and container images. For example, by mounting `/var/lib/containers` in a separate partition, the kubelet separately monitor `/var/lib/containers` as the `imagefs` directory and the root file system as the `nodefs` directory.
 
-[WARNING]
+[IMPORTANT]
 ====
-The use of custom partitions could result in those partitions not being monitored by {product-title} or alerted on. If you are overriding the default partitioning, see link:https://access.redhat.com/articles/4766521[Understanding OpenShift File System Monitoring (eviction conditions)] for more information about how {product-title} monitors your host file systems.
+If you resized your disk size to host a larger file system, consider creating a separate `/var/lib/containers` partition. This is important for a disk formatted to `xfs`, where a high number of allocation groups might cause CPU time issues.
```

Comment:
```suggestion
If you have resized your disk size to host a larger file system, consider creating a separate `/var/lib/containers` partition. This is important for a disk formatted to `xfs`, where a high number of allocation groups might cause CPU time issues.
```
- Adding a present participle indicates more clearly that you're talking about a present situation, imo. WDYT?
- Per the ISG [avoid ambiguous pronoun references](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references); what does "This" refer to? Try to rephrase the last sentence to avoid a "this is" construction at all (see the ISG word usage guidance for "[there is, there are](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t)" for an example.